src/renderer/services/db/api/repository.ts

```diff
@@ -56,7 +56,18 @@ export abstract class Repository<T> {
 }
 
 export abstract class BulkRepository<T> extends Repository<T> {
-    public async getAll(): Promise<T[]> {
+    public async getAll(id = Buffer.alloc(0)): Promise<T[]> {
+        const key = encodeKey(this.bucket, id);
+        const lt = encodeKey(this.bucket, Buffer.concat([id, Buffer.alloc(96).fill(Buffer.from("z"))]));
```

Comment:
I don't think z is last character in ascii :)

Hint you could fill it with max integer that fits into 1 byte